Add META REFRESH tag for HTML-only redirect

EdgeCaseBerg opened the pull request:
Support browsers that have javascript off via html refresh tags. And ask users to turn on JS so the documentation will work.

EdgeCaseBerg committed:
Add META REFRESH tag for HTML-only redirect

Support browsers that have javascript off via html refresh tags.

EdgeCaseBerg committed:
Add note about enabling Javascript

The refresh will send them to the page, but they're not going to be able
to use the documentation unless they turn on Javascript!

ssimeonov commented on index.html:
```diff
@@ -3,6 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <title>Project Documentation</title>
+	<meta http-equiv="refresh" content="5;URL='https://swoop-inc.github.io/scala-sugar/latest/api/'" />
```
Inconsistent whitespace.

EdgeCaseBerg replied:
fixed

ssimeonov commented on index.html:
```diff
@@ -15,6 +16,9 @@
     </script>
 </head>
 <body>
+<p>You will be redirected to the documentation below in 5 seconds. Note
```
The redirect code does not wait for 5 seconds so this message is a bit misleading.

EdgeCaseBerg replied:
The HTML Meta refresh waits for 5 second though. If you shut off java-script in your browser you'll see this behavior. I could put the `p` tag into a <noscript> tag though for clarity

EdgeCaseBerg replied:
added noscript tag for clarity. the p tag should only display when javascript is enabled.

EdgeCaseBerg committed:
Add noscript tag and fix indentation

noscript tag will ensure that the message about redirecting (via meta
tag) after 5 seconds will only show if the redirect will only take 5
seconds itself.

EdgeCaseBerg committed:
Fix whitespace in header

ssimeonov commented:
:+1:
